pkg/cloud/scope/machine.go

```diff
@@ -371,6 +377,10 @@ func (m *MachineScope) IsEKSManaged() bool {
 	return m.InfraCluster.InfraCluster().GetObjectKind().GroupVersionKind().Kind == ekscontrolplanev1.AWSManagedControlPlaneKind
 }
 
+func (m *MachineScope) IsControlPlaneExternallyManaged() bool {
```

Comment:
A bit confusing on why this is needed, the control plane reference can be anything, including a new CRD; why do we need to mark it as externally managed?